src/Layout.scss

```diff
@@ -53,12 +65,18 @@
   .logo-area {
     float: left;
     width: 320px;
+    @media screen and (min-width:1100px){
```

Comment:
For media queries we use Bootstrap4's built-in media queries. See: 
https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints and use mixins like @include media-breakpoint-down(xs) { ... } as needed.